jekyll/_includes/header.html

```diff
@@ -1,3 +1,4 @@
+<div class="sunset-banner">Photon has been retired since our <a title="June 2021 Release Notes" href="https://www.mozilla.org/en-US/firefox/89.0/releasenotes/?source=techstories.org">design refresh release in June 2021</a>. This project is no longer maintained and will no longer be updated.</div>
```

Comment:
```suggestion
<div class="sunset-banner">Photon has been retired since our <a title="June 2021 Release Notes" href="https://www.mozilla.org/en-US/firefox/89.0/releasenotes/">design refresh release in June 2021</a>. This project is no longer maintained and will no longer be updated.</div>
```
looks like the wrong source attribute, I removed it for brevity

Reply:
Thanks @jens1o! I have a few more changes to commit but will include the source fix.